app/features/build_runner/fastlane_build_runner.rb

```diff
@@ -105,9 +105,10 @@ def run(completion_block: nil, &block)
         logger.info("fastlane run complete")
         logger.debug(build_output.join("\n").to_s)
 
-        # TODO: need real artifacts here, are they artifacts or artifact paths?
```

Comment:
looks like this is duplicate changes from your other PR around artifacts

Reply:
I don't know w** is happening with my git today